packages/react-core/src/components/Progress/examples/ProgressTitleOutsideOfProgressBar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListDescription,
+  Progress,
+  ProgressMeasureLocation
+} from '@patternfly/react-core';
+
+export const ProgressTitleOutsideOfProgressBar: React.FunctionComponent = () => (
+  <DescriptionList>
+    <DescriptionListGroup>
+      <DescriptionListTerm id="progress-label">Title outside of progress bar</DescriptionListTerm>
```

Comment:
More of an "in case" thing, but just to prevent any issues in the future with duplicate IDs:

```suggestion
      <DescriptionListTerm id="title-outside-progress-example-label">Title outside of progress bar</DescriptionListTerm>
```